src/js/actions/tools.js

```diff
@@ -779,8 +781,7 @@ define(function (require, exports) {
         }.bind(this), []);
     
         _vectorMaskHandler = function () {
-            var toolStore = this.flux.store("tool"),
-                vectorMode = toolStore.getVectorMode() || false;
+            var vectorMode = toolStore.getVectorMode() || false;
```

Comment:
Instead of doing `|| false` here it would be better to change the tool store so that the default value is `false` instead of `null`.